crypto/commitments/rsa_based.go

```diff
@@ -43,7 +42,7 @@ func GenerateRSABasedQOneWay(nBitLength int) (func(*big.Int) *big.Int, func(*big
 		return nil, nil, nil, nil, err
 	}
 	if q.Cmp(rsa.N) < 1 {
-		return nil, nil, nil, nil, fmt.Errorf("Q must be > N")
+		return nil, nil, nil, nil, fmt.Errorf("q must be > n")
```

Comment:
I would rather have the names of mathematical variables in the same case as they are in comments. Which means upper-case `Q` and `N` here.

Reply:
Huh, you're right. Done!